eo-maven-plugin/src/main/java/org/eolang/maven/tojos/ForeignTojos.java

```diff
@@ -146,6 +147,65 @@ public Collection<ForeignTojo> isNotDiscoveredYet() {
             .collect(Collectors.toList());
     }
 
+    /**
+     * Get the tojos that have corresponding xmir.
+     * @return The tojos.
+     */
+    public Collection<ForeignTojo> withXmir() {
+        return this.tojos.value().select(row -> row.exists(Attribute.XMIR.key()))
+            .stream()
+            .map(ForeignTojo::new)
+            .collect(Collectors.toList());
+    }
+
+    /**
+     * Get the tojos that have corresponding eo file.
+     * @return The tojos.
+     */
+    public Collection<ForeignTojo> withEo() {
+        return this.select(row -> row.exists(Attribute.EO.key()))
+            .stream()
+            .map(ForeignTojo::new)
+            .collect(Collectors.toList());
+    }
+
+    /**
+     * Get the tojos that have corresponding xmir_2.
+     * @return The tojos.
+     */
+    public Collection<ForeignTojo> withSecondXmir() {
+        return this.tojos.value().select(row -> row.exists(Attribute.XMIR_2.key()))
+            .stream()
+            .map(ForeignTojo::new)
+            .collect(Collectors.toList());
+    }
+
+    /**
+     * Get the tojos that do not have corresponding eo and xmir.
+     * @return The tojos.
+     */
+    public Collection<ForeignTojo> withoutEoAndXmir() {
```

Comment:
@volodya-lombrozo Can we make this name simpler?  According to: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html